crates/core/src/instance.rs

```diff
@@ -77,6 +77,11 @@ impl Instance {
         scheduler
             .every(1.second())
             .run(scheduled_jobs::create_timeout_callback(context.clone()));
+        scheduler
+            .every(5.seconds())
```

Comment:
Should we use constants here, rather than literal numbers?

Reply:
I started doing this, but I actually think it makes it less readable in this case. Those numbers are only used here and seeing them here gives much more context as they are next to the function that gets scheduled. What we could do (in another PR) is to pull all these constants out and into the config so we can alter them without rebuild. But there wasn't a strong pull for this yet.